src/screencast/pipewire_screencast.c

```diff
@@ -13,6 +13,35 @@
 #include "xdpw.h"
 #include "logger.h"
 
+static bool pwr_y_invert_buffer(struct pw_buffer *buffer) {
```

Comment:
TBH, I'd prefer to error out when receiving y-inverted buffer, at least for now. There's no known compositor using it right now.